docs/getting-started/cli.rst

```diff
@@ -93,6 +93,164 @@ The ``seals:`` prefix ensures that the seals package is imported and the environ
             demonstrations.n_expert_demos=50
 
 
+Train an expert and save the rollouts explicitly, then train a policy on the saved rollouts:
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+First, train an expert and save the demonstrations. 
+Note that by default these are saved in ``<log_dir>/rollouts/final.npz``.
+Where for this script by default ``<log_dir>`` is ``output/train_rl/<environment>/<timestamp>`` .  
+However, we can pass an explicit path as logging directory. 
+By default, this will use ``ppo``. 
+
+.. code-block:: bash
+
+        python -m imitation.scripts.train_rl with pendulum \
+                logging.log_dir=output/train_rl/Pendulum-v1/my_run  \
```

Comment:
May want to illustrate the `rollout_save_n_{timesteps,episodes}` flags here. Could also potentially mention `eval_policy` if you want to get rollouts from an already trained expert.

Reply:
both fixed in 4807a15920953cb21f1ff593d84cc1ae9871cf04 and ccfc99d18aa5a4243b9712d1066f7321c1d87f8c , respectively.